tools/webpack.config.prod.js

```diff
@@ -0,0 +1,99 @@
+'use strict';
+
+var version = require('./../package.json').version;
+var path = require('path');
+var webpack = require('webpack');
+var autoprefixer = require('autoprefixer');
+
+var rootPath = path.join(__dirname, '..');
+
+function root(p){
+  return path.join(rootPath, p);
+}
+
+var srcPath = root('src');
+var componentsPath = root('src/components');
+var sassPath = root('src/sass');
+var docsPath = root('src/docs');
+var fontsPath = root('src/fonts');
+
+
+var distPath = root('dist');
+
+module.exports = {
+  entry: [
+    './src/entry'
+  ],
+  output: {
+    path: distPath,
+    publicPath: '/dist',
+    filename: 'bundle.js'
+  },
+  plugins: [
+    new webpack.NoErrorsPlugin(),
```

Comment:
* **[OccurenceOrderPlugin](https://webpack.github.io/docs/list-of-plugins.html#occurenceorderplugin)** keep the order the modules.This make its ids predictable, reduces to total file size and is recommended. This also facilitates further debugging.
* **[DedupePlugin](https://webpack.github.io/docs/list-of-plugins.html#dedupeplugin)** removes duplicates of libraries and other stuffs from bundle.
* **[NoErrorsPlugin](https://webpack.github.io/docs/list-of-plugins.html#noerrorsplugin)** prevents the creation of files, if there were any errors during compile.